extensions/iterm/src/open-iterm-here.tsx

```diff
@@ -12,6 +12,10 @@ const getItemPath = (item: FileSystemItem) => {
   return stats.isDirectory() ? item.path : dirname(item.path);
 };
 
+interface Preferences {
+  windowOrTab: "new-window" | "new-tab";
+}
```

Comment:
You don't need to manually set preference types as this is autogenerated in `raycast-env.d.ts`